src/request/create/index.html

```diff
@@ -42,7 +42,7 @@
     <link rel="stylesheet" href="../../components/PassphraseInput.css">
     <link rel="stylesheet" href="../../components/RecoveryWords.css">
     <link rel="stylesheet" href="../../components/ValidateWords.css">
-    <link rel="stylesheet" href="./privacy-agent.css">
+    <link rel="stylesheet" href="../../components/PrivacyAgent.css">
```

Comment:
This is already fixed in Sven's PR.

Reply:
Or rather the CSS is already added some lines above.

Reply:
You mean #54 ?
Apparently removal of this file was forgotten there